### PR TITLE
Standardise all friend commands and add-plan command

### PR DESCRIPTION
**Pull request by victorlaiyeeteng**
**Standardised the outputs of all friend commands. 
For add-plan command**
1. Corrected missing arguments message output
2. Corrected error outputs

### CODE REVIEW

File: src/main/java/seedu/address/logic/Messages.java [hunk 1]
```diff
@@ -12,10 +12,14 @@
  * Container for user visible messages.
  */
 public class Messages {
-    public static final String MESSAGE_PERSON_DOES_NOT_EXIST = "Friend not in addressbook";
+    public static final String MESSAGE_PERSON_DOES_NOT_EXIST = "Friend not in FriendBook";
 
-    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
+    public static final String MESSAGE_UNKNOWN_COMMAND = "Invalid command. Here are all the valid commands:\n"
+            + "Friends: add-friend, edit-friend, delete-friend, find-friend, list-friend\n"
+            + "Plans: add-plan, edit-plan, complete-plan, uncomplete-plan, find-plan, list-plan\n"
```
Comment (jannnice) [inline src/main/java/seedu/address/logic/Messages.java:19]: missing delete-plan

**Review by jannnice — APPROVED** [review]
LGTM (other than the one comment above)

File: src/main/java/seedu/address/model/plan/PlanDateTime.java [hunk 2]
```diff
@@ -12,11 +12,13 @@
...
  */
 public class PlanDateTime {
-    public static final String MESSAGE_CONSTRAINTS =
-            "Date Time should be of the form YYYY-MM-DD-HH:MM";
+    public static final String MESSAGE_INVALID_DATETIME = "Date-Time given is invalid.\n";
 
-    public static final String FUTURE_MESSAGE_CONSTRAINT =
-            "Date Time should be in the future!";
+    public static final String MESSAGE_CONSTRAINTS = MESSAGE_INVALID_DATETIME + " "
+            + "Date Time must be in YYYY-MM-DD-HH:MM format";
```
Comment (Elijah5399) [inline src/main/java/seedu/address/model/plan/PlanDateTime.java:18]: Inconsistent with what you wrote in the UG, which was "Date-Time given is invalid. Date-Time must be in YYYY-MM-DD-HH:MM format" (missing hyphen)

**Review by Elijah5399 — COMMENTED** [review]
Just a typo in the code missing hyphens. Other than that, LGTM!

**Pull request merged**